pathutil/path_filter_test.go

```diff
@@ -1,15 +1,16 @@
-package fsfilter
+package pathutil_test
```

Comment:
package needs to be pathutil_test?

Reply:
In order to overcome pathutil - fileutil reference circle. 
The test requires fileutil, but fileutil includes pathutil.

Reply:
😢